OSDOCS-17075_d#Gateway API CQA (#115162)
openshift/openshift-docs: modules/nw-ingress-gateway-api-manage-succession.adoc

= Preparing for Gateway API management succession by the Ingress Operator

[role="_abstract"]

> Starting in {product-title} 4.19, the Ingress Operator manages Gateway API custom resource definitions (CRDs). Replace or remove existing non-compliant CRDs before you upgrade to prevent conflicts and to ensure update success.

Reviewer: ```suggestion
Prepare your cluster for Gateway API management succession by removing existing unsupported definitions and installing compliant resources. This ensures a seamless update to {product-title} 4.19 and prevents conflicts with the Ingress Operator.

Starting in {product-title} 4.19, the Ingress Operator manages Gateway API custom resource definitions (CRDs).  This lifecycle control blocks you from creating, updating, or deleting CRDs within the `gateway.networking.k8s.io` API group.
```

> You can prepare your cluster for Gateway API management succession by removing existing unsupported definitions and installing compliant resources. This ensures a seamless update to {product-title} 4.19 and prevents conflicts with the Ingress Operator.

Reviewer: ```suggestion
```
see the previous edit for where this content is suggested to move

> Starting in {product-title} 4.19, the Ingress Operator manages the lifecycle of any Gateway API  CRDs. This lifecycle control blocks you from creating, updating, or deleting CRDs within the `gateway.networking.k8s.io` API group.

Reviewer: ```suggestion
```
redundant with the updated abstract
